bodhi/server/templates/update.html

```diff
@@ -162,6 +162,9 @@
         // seem to break anything
         var arch = result.data.arch;
         var item = result.data.item;
+        if (item === undefined){
```

Comment:
I think it would be better to do
```
if (item === undefined && testcase == 'org.centos.prod.ci.pipeline.complete') {
```
it would then be clear that this is for this specific testcase.